pkg/ovs/ofctl/ofctl.go

```diff
@@ -21,46 +21,40 @@ import (
 	"strings"
 )
 
-type OfctlClient struct {
+type ofctlClient struct {
```

Comment:
did you forget to check-in a new file? I don't see the `OfctlClient` interface definition

Reply:
Yes, I missed 3 new files in the last update. Already added them.